qiskit/converters/circuits_to_qobj.py

```diff
@@ -28,8 +24,8 @@ def circuits_to_qobj(circuits, user_qobj_header=None, run_config=None,
         circuits (list[QuantumCircuits] or QuantumCircuit): circuits to compile
         user_qobj_header (QobjHeader): header to pass to the results
         run_config (RunConfig): RunConfig object
-        qobj_id (int): identifier for the generated qobj
 
+        qobj_id (int): TODO: delete after qiskit-terra 0.8
         backend_name (str): TODO: delete after qiskit-terra 0.8
         config (dict): TODO: delete after qiskit-terra 0.8
         shots (int): TODO: delete after qiskit-terra 0.8
```

Comment:
This whole file is being deprecated so I think there can be one `TODO: delete after 0.8` for the whole file.

Reply:
I agree.